src/modules/support/login.ts

```diff
@@ -0,0 +1,92 @@
+import axios from 'axios'
+import chalk from 'chalk'
+import * as inquirer from 'inquirer'
+import * as jwt from 'jsonwebtoken'
+import { prop } from 'ramda'
+import { getAccount, getToken, getWorkspace, saveAccount, saveToken, saveWorkspace } from '../../conf'
+import * as env from '../../env'
+import log from '../../logger'
+
+const getAvailableRoles = async (region: string, token: string, supportedAccount: string): Promise<string[]> => {
+  const response = await axios.get(
+    `http://support-authority.vtex.${region}.vtex.io/${getAccount()}/${getWorkspace()}/${supportedAccount}/roles`,
+    {
+      headers: {
+        'Authorization': token,
```

Comment:
Should this have an explicit authorization type?

Reply:
No need. We only have 1 type and has to be the default as many people do not add an explicit type already.